runtime/src/cost_tracker.rs

```diff
@@ -114,7 +126,12 @@ impl CostTracker {
         (costliest_account, costliest_account_cost)
     }
 
-    fn would_fit(&self, keys: &[Pubkey], cost: &u64) -> Result<(), CostTrackerError> {
+    fn would_fit(
+        &self,
+        keys: &[Pubkey],
+        cost: &u64,
+        account_data_len: u64,
+    ) -> Result<(), CostTrackerError> {
```

Comment:
nit: maybe use same type for `cost` and `account_data_len` , either both ref or neither ref.